portfolio/src/main/webapp/quiz.js

```diff
@@ -2,40 +2,125 @@
  * Question data type.
  */
 class Question {
-  constructor(question, answers, correct) {
+  /**
+   * @param {string} question The question that shall be posed
+   * @param {Array<string>} answers The potential responses to the question
+   * @param {number} correctAnswerIndex The index in answers that leads to the correct response
+   */
+  constructor(question, answers, correctAnswerIndex) {
+    /**
+     * @private @const {string}
+     */
```

Comment:
curious - is JSDoc required inside of the constructor?

Reply:
That's what I gathered from the [style guide](https://google.github.io/styleguide/jsguide.html#jsdoc-property-comments).

Reply:
Ah - i see. they can be optionally omitted for private properties. go ahead and leave as is.